app/views/certificates/a_level/_achievement_list.html.erb

```diff
@@ -12,7 +12,7 @@
 
       <div class='ncce-activity-list__button-wrapper'>
         <%= link_to t('.all_courses_button'),
-          courses_path(level: 'Key stage 5'),
+          courses_path(level: 'Post 16'),
```

Comment:
This doesn't look quite right as there will probably be courses in that link that aren't in the programmes_activities table. IMO it should match what the user gets when the pick the 'A Level' course filter themselves.
<img width="512" alt="Screenshot 2023-09-19 at 16 04 31" src="https://github.com/NCCE/teachcomputing.org/assets/2813357/ae45b014-c21e-403c-b2b4-84c8e4986a69">

Reply:
Oh dear, even the filter seems to depend on Smart Connector data. That said, in my opinion the Figma was misleading. I suggest you merge now and revisit when the courses are configured, but for consistency at least, I think the certificate category is more likely to be consistent with courses that will actually show up in the user's A Level dashboard as enrolled/completed.

Reply:
I do agree